package.json

```diff
@@ -14,12 +14,14 @@
     "lint:fix": "./node_modules/.bin/eslint . --fix"
   },
   "dependencies": {
-    "bunyan": "^1.8.9",
-    "shr-models": "^5.2.1"
+    "bunyan": "^1.8.9"
   },
   "devDependencies": {
     "chai": "^3.5.0",
     "eslint": "^3.6.1",
     "mocha": "^3.2.0"
```

Comment:
shr-test-helpers doesn't actually appear to use mocha, I think that we can drop the dependency entirely.

Reply:
OK.  I'll buy that.